frontend/www/js/omegaup/contest/edit.ts

```diff
@@ -65,12 +65,29 @@ OmegaUp.on('ready', () => {
           })
           .catch(ui.apiError);
       },
-      refreshProblems: (): void => {
+      refreshProblems: (problemAdded: boolean): void => {
         api.Contest.problems({
           contest_alias: payload.details.alias,
         })
           .then((response) => {
             contestEdit.problems = response.problems;
+            if (problemAdded) {
+              if (
+                contestEdit.problems[contestEdit.problems.length - 1].languages
```

Comment:
los usuarios pueden elegir agregar el problema no-al-final, entonces esto a veces fallaría. lo ideal es checar todos los lenguajes de todos los problemas:

```suggestion
                contestEdit.problems.some((problem) => problem.languages.split(',').includes('cat'))
```